logpyle/__init__.py

```diff
@@ -189,11 +190,11 @@ def __init__(self, names: List[str], units: Optional[List[str]] = None,
         self.names = names
 
         if units is None:
-            units = len(names) * [""]
+            units = len(names) * [None]
```

Comment:
maybe try:

```suggestion
            units: List[None] = len(names) * [None]
```

and 

```
units: Optional[List[Optional[str]]] = None,
```

above.

Reply:
Another possibility:

```
if units is None:
    self.units = len(names) * [None]
else:
    self.units = units
```

Reply:
I was able to effectively do the second possibility with the use of casting. (does nothing at runtime)